### PR TITLE
Update EULA service error message.

### PR DESCRIPTION
**Pull request by forvalho**
I AM HELPING!

### CODE REVIEW

File: www/app/common/update-eula.service.js [hunk 1]
```diff
@@ -45,8 +45,8 @@
           },
         })
         .catch(function() {
-          console.log('error submitting eula');
+          console.log('Error submitting EULA!');
         });
     }
   }
-})();
```
Comment (filipesilva) [inline www/app/common/update-eula.service.js:52]: y u remove newline? :<

**Comment by lnaia** [comment]
I AM TROLLING!

File: www/app/common/update-eula.service.js (hunk 1 above)
Comment (lnaia) [inline www/app/common/update-eula.service.js:52]: wow
such change
much bracket
so clean

Comment (forvalho) [inline www/app/common/update-eula.service.js:52]: Github did it. It's good practice to have an empty line at EOF anyway! http://stackoverflow.com/a/2287990

Comment (lnaia) [inline www/app/common/update-eula.service.js:52]: y I always force it

Comment (filipesilva) [inline www/app/common/update-eula.service.js:52]: ok i will accept but pls add tests for newline next time

**Pull request merged**